_overviews/scala3-book/domain-modeling-oop.md

```diff
@@ -89,12 +89,11 @@ When designing software in Scala, it’s often helpful to only consider using cl
 NOTE: I think “leaves” may technically be the correct word to use, but I prefer “leafs.”
 {% endcomment %}
 
-```text
-traits               T1   T2           ...  T3
-composed traits      S extends T1, T2  ...  S extends T2, T3
-classes              C extends S, T3
-instances            new C
-```
+| Traits          | `T1`, `T2`, `T3`
```

Comment:
At the moment I can’t run Jekyll on my computer to render this, but if that table syntax works, this is fine by me.

Reply:
Sorry, I should have included a screenshot. It looks like this:

![Screenshot from 2021-06-14 10-27-37](https://user-images.githubusercontent.com/332812/121862565-356b1580-ccfb-11eb-98ec-df2b498eb5d0.png)

Reply:
No problem on this. I just had a problem with Jekyll and its tools, and I didn’t have time to look into how to fix it.